addons/spreadsheet/static/src/chart/odoo_menu/figure_component.js

```diff
@@ -12,26 +12,19 @@ patch(spreadsheet.components.FigureComponent.prototype, {
     },
     async navigateToOdooMenu() {
         const menu = this.env.model.getters.getChartOdooMenu(this.props.figure.id);
-        if (!menu) {
-            throw new Error(`Cannot find any menu associated with the chart`);
-        }
-        if (!menu.actionID) {
-            this.notificationService.add(
-                _t(
-                    "The menu linked to this chart doesn't have an corresponding action. Please link the chart to another menu."
-                ),
-                { type: "danger" }
-            );
-            return;
-        }
-        await this.actionService.doAction(menu.actionID);
+        await navigateToOdooMenu(menu, this.actionService, this.notificationService);
     },
     get hasOdooMenu() {
         return this.env.model.getters.getChartOdooMenu(this.props.figure.id) !== undefined;
     },
     async onClick() {
-        if (this.env.isDashboard() && this.hasOdooMenu) {
-            this.navigateToOdooMenu();
+        try {
```

Comment:
what kind of error is this try/catch supposed to catch ? is it from `this.navigateToOdooMenu()` ?

EDIT: wait, this method is no longer used, is it ?

Reply:
No longer used indeed